context/build-utils/webpack.common.js

```diff
@@ -3,11 +3,11 @@ const { resolve } = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
 const config = {
-  entry: './app/static/js/index.jsx',
+  entry: { main: './app/static/js/index.jsx' },
   output: {
     path: resolve('./app/static/public'),
-    filename: 'bundle.js',
-    publicPath: resolve('../app/static/public'),
+    publicPath: `${resolve('/static/public/')}/`,
```

Comment:
The final "/" probably is required? Would it just be an error if you gave a directory name without the final "/"?

Reply:
Yeah, not including the additional / was producing an error for whatever reason.